tools/doc/html.mjs

```diff
@@ -325,7 +326,7 @@ export function preprocessElements({ filename }) {
           // Insert div with prefix and number
           node.children.unshift({
             type: 'html',
-            value: `<div class="api_stability api_stability_${number}">` +
+            value: `<div class="api_stability api_stability_${parseInt(number)}">` +
```

Comment:
parseInt will only grab the numeric pieces of "number" right?

Reply:
Yes, in the regex that the part for `number` is `/\d(?:\.\d)?` meaning that the `number` matched value can be a single digit optionally followed by a dot followed by another digit, `parseInt` run on such numbers/strings always returns the integer part of the number:
![Screenshot at 2025-06-22 18-50-14](https://github.com/user-attachments/assets/a7536068-b099-429d-a303-3a1ab5017238)

And that's the part we need based on these are the css styles:
https://github.com/nodejs/node/blob/d7becc58750096b711c55a378a55f596a45f3ce1/doc/api_assets/style.css#L324-L338

Reply:
I used the same trick in the web generator :-)

Reply:
No, not until it finds something non-numeric. It parses the float since it's a valid number, and flattens it to an integer.